### ISSUE TITLE
TMUX session does not end when node crashes

### ISSUE THREAD

**Issue by CAJan93** [risingwavelabs/risingwave#6467]
### To Reproduce

Write a `panic` in one of your nodes, e.g. `meta`

Watch the nodes `risedev lsw` [see PR](https://github.com/risingwavelabs/risingwave/pull/6445)

```
risedev lsw
1: grafana
2: prometheus
3: etcd-2388
4: etcd-12388
5: etcd-1234 # THIS NODE CRASHES
6: meta-node-5690
7: meta-node-15690
9: compute-node-5688
10: frontend-4566
```
 

### Expected behavior

The tmux session of the crashed node should exit. 

```
risedev lsw
1: grafana
2: prometheus
3: etcd-2388
4: etcd-12388
6: meta-node-5690
7: meta-node-15690
9: compute-node-5688
10: frontend-4566
```

**Comment by skyzh** [comment]
Expected behavior. See run_command.sh.

**Comment by fuyufjh** [comment]
Shall we close it?

**Comment by skyzh** [comment]
I think we should change the implementation of risedev l.

**Comment by CAJan93** [comment]
I agree @skyzh. Maybe this is expected behaviour, but from a user perspective it was surprising to see nodes still being listed, even though they crashed.

**Comment by skyzh** [comment]
Yep, let's see how to fix this...

**Comment by CAJan93** [comment]
> Yep, let's see how to fix this...

Thank you for taking the task 👍

**Comment by CAJan93** [comment]
Is there an update regarding this issue? Do we want to keep it around or should we close it?

**Comment by xxchan** [comment]
No one seems to be interested in fixing this (Actually I have never used `risedev ls` 🤣
